util/feeding.py

```diff
@@ -44,7 +44,8 @@ def samples_to_mfccs(samples, sample_rate):
 def audiofile_to_features(wav_filename):
     samples = tf.io.read_file(wav_filename)
     decoded = contrib_audio.decode_wav(samples, desired_channels=1)
-    features, features_len = samples_to_mfccs(decoded.audio, decoded.sample_rate)
+    audio = pertub(decoded.audio)
```

Comment:
This needs to be behind a check for the flag you added, but the check can't be here, it needs to be in the caller of `create_dataset` otherwise you'll also augment the validation and test sets.